scripts/config/git-config.rb

```diff
@@ -0,0 +1,2 @@
+system "git config --global user.email '84353015+secman-bot@users.noreply.github.com'"

```

Comment:
are you sure about the email @david-tomson

Reply:
yes I'm sure